sequencer/src/options.rs

```diff
@@ -64,6 +64,16 @@ pub struct Options {
     #[clap(short, long, env = "ESPRESSO_SEQUENCER_CONFIG_PATH")]
     pub config_path: Option<PathBuf>,
 
+    /// The amount of time to wait between each request to the HotShot
+    /// consensus or DA web servers during polling, in milliseconds.
+    #[clap(
+        short,
+        long,
+        env = "ESPRESSO_SEQUENCER_WEBSERVER_POLL_INTERVAL_MS",
+        default_value = "100"
+    )]
+    pub webserver_poll_interval_ms: u64,
```

Comment:
Nit: I prefer to make this `Duration`, like [in the orchestrator](https://github.com/EspressoSystems/espresso-sequencer/blob/main/sequencer/src/bin/orchestrator.rs#L25-L26), parsed  [like this](https://github.com/EspressoSystems/espresso-sequencer/blob/main/sequencer/src/bin/orchestrator.rs#L116-L122). This gives us type safety in the code and lets users/terraform write things like `100ms` or `1s`, which can be easier to read and understand